### PR TITLE
Johnny

### PR DESCRIPTION
**Pull request by marantha**
taking code from bb

### CODE REVIEW

File: components/Fridge.js [hunk 1]
```diff
@@ -1,79 +1,130 @@
...
+  Button,
+  Alert,
+} from "react-native";
+//import { get } from "react-native/Libraries/Utilities/PixelRatio";
 
 const styles = StyleSheet.create({
   container: {
-   flex: 1,
-   paddingTop: 22
+    flex: 1,
```
Comment (marantha) [inline components/Fridge.js:17]: why did you put space?

File: components/Fridge.js [hunk 2]
```diff
@@ -1,79 +1,130 @@
...
 
-const Item = ({ title }) => (
-  <View style={styles.item}>
-    <Text style={styles.title}>{title}</Text>
-  </View>
-);
+export const getTodaysDate = () =>{
+  var today = new Date();
+  var dd = String(today.getDate()).padStart(2, '0');
+  var mm = String(today.getMonth() + 1).padStart(2, '0'); //January is 0!
```
Comment (marantha) [inline components/Fridge.js:54]: remove comment

File: components/Fridge.js (hunk 1 above)
Comment (johnnypark777) [reply components/Fridge.js:17]: cuz I felt like it

**Review by johnnypark777 — APPROVED** [review]
looks good to me!

**Pull request merged**